cmd/aries-agent-rest/startcmd/start.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/rs/cors"
 	"github.com/spf13/cobra"
 
+	couchdbstore "github.com/hyperledger/aries-framework-go-ext/component/storage/couchdb"
+	"github.com/hyperledger/aries-framework-go-ext/component/storage/mysql"
```

Comment:
these are actually intended to be removed rather than referencing the ext repo.

Reply:
Removed!